### ISSUE TITLE
litematica combined with optifine doesnt work properly

### ISSUE THREAD

**Issue by beware08** [maruohon/litematica#247]
When you create a placement while also having optifine loaded, the mod goes crazy and the schematic isnt usable anymore

**Comment by vacla** [comment]
===================================
 Litematica renders weird triangles, how to fix? 
===================================
 - Disable shaders in Optifine, and then restart the game.
 - You may need to disable Render Regions and Fast Render as well to fix other rendering issues.

**Comment by JxM-92** [comment]
so litematica and shaders hate each other?

**Comment by vacla** [comment]
What optifine does i believe

**Comment by SkillnoobHD** [comment]
I dont think litematica was ever deisgned to be compatible with optifine.
Try sodium or canvas instead and they dont break the litematica rendering.

**Comment by Nopotato** [comment]
I use frabrick, optifabric, litematica with 1.16.3, and sometimes it works fine and beautifully with shaders on, and other times it doesn't work at all. I have tried disabling render regions and fast render, but have not been able to get it to work again. I am not sure what causes the rendering issues, seeing as sometimes it just works fine.

**Comment by vacla** [comment]
Shaders disabled and restarted the game? @Nopotato

**Comment by Nopotato** [comment]
> Shaders disabled and restarted the game? @Nopotato

Yes, disabling the shaders and restarting does not work, although sometimes (maybe 2/10 attempts) it does work, unless it was just a coincidence. For now I have chosen to remove the optifine set, to use litematica.

**Comment by Nopotato** [comment]
@maruohon Wow, that's awesome! I had tried to update my custom modpack but kept running into issues so i rolled back to an older instance until I had a combo that worked though I am prone to user error. Either way this one you linked me now works on my quick 1 minute test so I am very happy.
